pom.xml

```diff
@@ -51,8 +51,8 @@
         <confluent.version>5.0.1-SNAPSHOT</confluent.version>
         <easymock.version>3.6</easymock.version>
         <exec-maven-plugin.version>1.5.0</exec-maven-plugin.version>
-        <findbugs-maven-plugin.version>3.0.5</findbugs-maven-plugin.version>
-        <findbugs-annotations.version>3.0.1</findbugs-annotations.version>
+        <spotbugs-maven-plugin.version>3.1.6</spotbugs-maven-plugin.version>
+        <spotbugs-annotations.version>3.1.6</spotbugs-annotations.version>
```

Comment:
There was a regression in 3.1.6 that caused issue when building Kafka. 3.1.5 was fine. Do we want to be conservative here too?

Reply:
What was the issue? as long as we have a working build, I don't think it matters to be on the newer version.

Worst case, we just reduce the version if we hit the same problem.

Reply:
It caused a stacktrace during the build. Since you used muckrake to test most projects, maybe that's fine. In any case, 3.1.7 will be out soon with a fix for the issue and Java 11 support, so hopefully we migrate to it before anyone can run into the problem while building locally.